dtfs-central-api/api-tests/v1/utilisation-reports/post-upload-utilisation-report.api-test.ts

```diff
@@ -15,40 +18,59 @@ import { testApi } from '../../test-api';
 import { SqlDbHelper } from '../../sql-db-helper';
 import { mongoDbClient } from '../../../src/drivers/db-client';
 import { wipe } from '../../wipeDB';
-import { aUtilisationReportRawCsvData, aPortalUser } from '../../../test-helpers';
+import { aUtilisationReportRawCsvData, aPortalUser, aFacility } from '../../../test-helpers';
 import { PostUploadUtilisationReportRequestBody } from '../../../src/v1/controllers/utilisation-report-service/post-upload-utilisation-report.controller';
 
 console.error = jest.fn();
 
 const getUrl = () => '/v1/utilisation-reports';
 
 describe(`POST ${getUrl()}`, () => {
-  const reportId = 12;
+  const REPORT_ID = 12;
+  const FACILITY_ID = '123456789';
 
-  const portalUser = aPortalUser();
-  const portalUserId = portalUser._id.toString();
+  const PORTAL_USER = aPortalUser();
+  const PORTAL_USER_ID = PORTAL_USER._id.toString();
```

Comment:
Can we stick to camelCase for all these consts please

Otherwise this will be the only test with a different convention.